.snyk

```diff
@@ -1,35 +1,22 @@
 # Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
 version: v1.22.1
 # ignores vulnerabilities until expiry date; change duration by modifying expiry date
 ignore:
   SNYK-JS-JSONWEBTOKEN-3180022:
     - '*':
-        reason: notifications-node-client needs to be upgraded to latest version
-        expires: 2024-09-15T00:00:00.000Z
-        created: 2023-09-15T13:10:47.350Z
+        reason: notifications-node-client needs to be upgraded in HOF to latest version
+        expires: '2024-09-15T00:00:00.000Z'
+        created: '2023-09-15T13:10:47.350Z'
   SNYK-JS-SEMVER-3247795:
     - '*':
-        reason: notifications-node-client needs to be upgraded to latest version
-        expires: 2024-09-15T00:00:00.000Z
-        created: 2023-09-15T13:10:47.350Z
-  SNYK-JS-REQUEST-3361831:
-    - '*':
-        reason: HOF framework needs updating to remove request.js
-        expires: 2024-03-01T00:00:00.000Z
-        created: 2023-08-09T13:22:47.350Z
-  SNYK-JS-TOUGHCOOKIE-5672873:
-    - '*':
-        reason: HOF framework needs updating to remove request.js of which this is a dependency
-        expires: 2024-03-01T00:00:00.000Z
-        created: 2023-08-09T13:22:47.350Z
+        reason: notifications-node-client needs to be upgraded in HOF to latest version
+        expires: '2024-09-15T00:00:00.000Z'
+        created: '2023-09-15T13:10:47.350Z'
   SNYK-JS-USERAGENT-174737:
     - device > useragent:
         reason: No available patch currently in circulation
         expires: '2024-11-01T17:02:21.865Z'
-  SNYK-JS-REQUEST-1314897:
-    - hof > request:
-        reason: Need to replace request in HOF
-        expires: '2024-09-18T14:58:46.388Z'
```

Comment:
keep this in - you haven't replaced request in this branch so it will fail drone if this is not included